src/accelerate/accelerator.py

```diff
@@ -285,8 +284,8 @@ def __init__(
         if os.environ.get("ACCELERATE_USE_FSDP", "false") == "true" or isinstance(
             fsdp_plugin, FullyShardedDataParallelPlugin
         ):
-            if is_torch_version("<", "1.12.0"):
-                raise ValueError("FSDP requires PyTorch >= 1.12.0")
+            if is_torch_version("<", "2.0.1"):
+                raise ValueError("FSDP requires PyTorch >= 2.0.1")
```

Comment:
A thought I had on these, to help users out perhaps we should give these errors context of what specific accelerate version they should be using, if they want to use less than 2.0 (and for any other deps) so that they don't always have to be using the latest version, and know when they should pin things to? (And make it more stable in that regard)

Reply:
FSDP is highly experimental as it is and keeps evolving with every release, for FSDP sadly i can't see a way for users to get the latest and robust features without upgrading Torch

Reply:
All good!